tests/integration/tables/rpm_packages.cpp

```diff
@@ -22,26 +24,31 @@ class rpmPackages : public testing::Test {
 };
 
 TEST_F(rpmPackages, test_sanity) {
-  // 1. Query data
-  auto const data = execute_query("select * from rpm_packages");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See helper.h for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidationMap row_map = {
-  //      {"name", NormalType}
-  //      {"version", NormalType}
-  //      {"release", NormalType}
-  //      {"source", NormalType}
-  //      {"size", IntType}
-  //      {"sha1", NormalType}
-  //      {"arch", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+  auto rows = execute_query("select * from rpm_packages");
+  if (rows.size() > 0) {
+    ValidationMap row_map = {{"name", NonEmptyString},
+                             {"version", NormalType},
+                             {"release", NormalType},
+                             {"source", NormalType},
+                             {"size", IntType},
+                             {"sha1", NonEmptyString},
+                             {"arch", NonEmptyString},
+                             {"epoch", IntType},
+                             {"install_time", IntType}};
+
+    validate_rows(rows, row_map);
+
+    if (isPlatform(PlatformType::TYPE_LINUX)) {
```

Comment:
@Smjert, should we create a small abstraction/helper method that takes a `table_name` and `row_map` and constructs the query, adds the additional columns, and calls `validate_rows`?

Reply:
Something like `validate_container_rows`?
I would still keep the isPlatform if outside of it though; what you think?

Reply:
Love it